src/react-components/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nypl/design-system-react-components",
-  "version": "0.0.13",
+  "version": "0.0.14",
```

Comment:
Did you mean to change the version in this file?

Reply:
Oh ugh I think this was merge funk.  But ... I'd love your thoughts on how to do this -- 
So the version was bumped for release (https://www.npmjs.com/package/@nypl/design-system-react-components) 
Should I then push that bump to development directly?

Reply:
I don't think there's a git flow in the readme but that'd be good to have and follow. I normally do work on a feature or development branch and then merge _that_ branch into `master`. Final changes go into `master` are then published. Is it normal for this flow to have a release and then merge back down to the development branch? Why?

Reply:
Wait, your `development` branch _is_ your production branch (currently). I think this should be documented because it seems confusing now. I know this is going through active development, but I don't know why you would want to make a release on a branch that's not stable, and then push that change into the stable branch.

Reply:
It's not normal, we just kind of started releasing things "as a test" and then ... before we realized, it's now in a state that definitely can be called "production".  And I agree it needs a real git flow and process because it's in that state now.

Reply:
Regardless, this isn't the place to discuss this, so let's do the broken stuff for now.  I've filed this guy: https://github.com/NYPL/nypl-design-system/issues/81 I would like to get it figured out sooner rather than later ...